blog/docs/events/virtual-office-hour-12-2022.md

```diff
@@ -0,0 +1,52 @@
+---
+
+## First 🌱 Knative Virtual Office Hour / Contrib Fest
+
+We think that there should be an easy way for new Knative users and contributors to get started with our project. Holding virtual office hours for questions and help with using or contributing to Knative should help to get started with using code and contributing.
+
+During the recent KubeCon NA 2022 in Detroit we had Knative Day and ContribFest that showed that there are people interested in using Knative or contributing to it. However, today we do not have any ongoing online meetings for people to just show up. They can join Slack but that is very asynchronous and joining yet another Slack and then reading channels, figuring where to ask your questions, etc. feels very daunting,
+
+Therefore, we are organizing the first Knative Virtual Office Hour / Contrib Fest in 2022.
+
+**When:** [December 9 (Friday) 11:30 AM US Eastern](https://dateful.com/convert/utc?t=430pm&d=2022-12-09)
+**How to join:** Zoom link will be posted in Slack and knative-user mailing list
+
+**Who should join?**
+
+- You are curious about Knative and you are wondering where to start
+- You would like to contribute to Knative but do not know where to start
+- You have problems with Knative and asking in Slack or Github issues did not get you what you wanted
+- You have great idea for Knative but yo do not want where to start
+- You want to get to know how Knative works inside and want to contribute to open source project
+
+**Possible topics for Contib fest:**
+
+- See [slides form KubeCon about Knative ContribFest](https://docs.google.com/presentation/d/1hnPHtbjLs3KMe-N62NhQeVtjH-p-obyI/edit?usp=sharing&ouid=112051875307901839973&rtpof=true&sd=true)
+- Browse good first issues:
+  - [client](https://github.com/knative/client/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
+  - [docs](https://github.com/knative/docs/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
+  - [eventing](https://github.com/knative/eventing/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
+  - [func](https://github.com/knative/func/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
+  - [serving](https://github.com/knative/serving/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fgood-first-issue)
+- Browse eventing sources in knative-sandbox to see what issues could be fixed of features added such as
+  - [eventing-couchdb](https://github.com/knative-sandbox/eventing-couchdb)
+  - [eventing-github](https://github.com/knative-sandbox/eventing-couchdb)
+  - [eventing-gitlab](https://github.com/knative-sandbox/eventing-couchdb)
+  - [eventing-kogito](https://github.com/knative-sandbox/eventing-couchdb)
+  - [eventing-redis](https://github.com/knative-sandbox/eventing-couchdb)
+  - Other [eventing sources and sandbox projects](https://github.com/knative-sandbox/)
+
+No prior knowledge of Knative is required.
+
+Hope to see you there!
```

Comment:
I think the feedback from the Maintainer Meetup was that we didn't have a community <-> contributors dialogue; I'd like to focus this more on that dialogue than on immediately moving folks into contributing.

Reply:
Yes. When we have the blog post published I think we should send email to knative-users to invite them to the office hour for dialogue and for us to listen.